subworkflows/local/pairalign_m2o/main.nf

```diff
@@ -85,12 +114,7 @@ workflow PAIRALIGN_M2O {
         .mix(ALIGNMENT_SPLIT_O2O.out.multiqc.collect{ it[1]} )
     m2o = ALIGNMENT_LASTAL_M2O.out.maf
     o2o = ALIGNMENT_SPLIT_O2O.out.maf
-    versions = Channel.empty()
-        .mix(     ALIGNMENT_LASTDB.out.versions)
-        .mix(      ALIGNMENT_TRAIN.out.versions)
-        .mix( ALIGNMENT_LASTAL_M2O.out.versions)
-        .mix(  ALIGNMENT_SPLIT_O2O.out.versions)
-        .mix(ALIGNMENT_DOTPLOT_O2O.out.versions)
+    versions = ch_versions                     // channel: [ versions.yml ]
```

Comment:
```suggestion
    versions = ch_versions
```

Reply:
I would like to keep this one as it comes from the `TEMPLATE` branch.